.github/workflows/build-wheels.yml

```diff
@@ -9,328 +9,59 @@ on:
-  # This means we have to write our own building and testing steps, but in a
-  # way it's simpler, because we don't need to care about configuring
-  # multibuild ourselves.
-  # Second, the syntax to enable virtual environments, etc. is different.
-  #
-  build_windows:
-    timeout-minutes: 35
-    runs-on: windows-latest
-    defaults:
-      run:
-        shell: bash
-
-    needs: [linters]
-
     strategy:
       fail-fast: false
       matrix:
-        include:
-        - python-version: "3.8"
-          build-depends: numpy==1.17.3
-
-        - python-version: "3.9"
-          build-depends: numpy==1.19.3
-
-        - python-version: "3.10"
-          build-depends: numpy==1.22.2 scipy==1.8.0
-        
-        - python-version: "3.11"
-          build-depends: numpy==1.23.2 scipy==1.9.2
-
-    env:
-      SKIP_NETWORK_TESTS: 1
-      TEST_DEPENDS: pytest mock testfixtures
-      BUILD_DEPENDS: ${{ matrix.build-depends }}
-
+        os: [ubuntu-20.04, windows-2019, macos-11]
     steps:
-    - uses: actions/checkout@v3
-      with:
-        submodules: recursive
-        fetch-depth: 0
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v4
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install virtualenv
 
-    - name: Build Wheel
-      run: |
-        echo ::group::Set up dependencies
-        python --version
-        python -c "import struct; print(struct.calcsize('P') * 8)"
-        python -m pip install -U pip setuptools wheel wheelhouse_uploader ${{ env.BUILD_DEPENDS }}
-        echo ::endgroup::
-        echo ::group::Build wheel
-        python setup.py bdist_wheel
-        echo ::endgroup
-        echo ::group::Install run
-        ls dist
-        python continuous_integration/install_wheel.py
-        echo ::endgroup::
-        #
-        # For consistency with the multibuild step.  The wheel uploader expects
-        # the wheels to be under wheelhouse.
-        #
-        mv dist wheelhouse
+      - name: Checkout
+        uses: actions/checkout@v3
 
-    - name: Prepare for testing
-      run: |
+      - name: Set up QEMU
+        if: runner.os == 'Linux'
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: all
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.12.0
+        env:
+          CIBW_ARCHS_LINUX: x86_64 aarch64
+          CIBW_ARCHS_MACOS: x86_64 arm64
+          CIBW_ARCHS_WINDOWS: AMD64 x86 ARM64
+          CIBW_BEFORE_BUILD: pip install numpy scipy
+          CIBW_SKIP: pp* cp36-* cp37-* *-win32 *_i686 *-musllinux_*
+          CIBW_TEST_COMMAND: pytest -rfxEXs --durations=20 --disable-warnings --showlocals --pyargs gensim
+          CIBW_TEST_REQUIRES: pytest testfixtures mock
+          CIBW_TEST_SKIP: cp38* cp39* cp310* *_aarch64 *_arm4 *_universal2
```

Comment:
```suggestion
          CIBW_TEST_SKIP: cp38* cp39* cp310* *_aarch64 *_arm64 *_universal2
```

Reply:
Can you please make this change @nikaro? I can't seem to do it from here.

Reply:
It looks like that removing my fork messed up things a bit. I cannot commit my suggestion either. And pushes to my branch are not reflected here...